src/main/java/org/broadinstitute/consent/http/db/UserDAO.java

```diff
@@ -171,12 +172,57 @@ void updateUser(@Bind("displayName") String displayName,
     @RegisterBeanMapper(value = User.class)
     @RegisterBeanMapper(value = UserRole.class)
     @UseRowReducer(UserWithRolesReducer.class)
-    @SqlQuery("SELECT du.*, r.name, ur.role_id, ur.user_role_id, ur.dac_id " +
+    @SqlQuery("SELECT du.*, r.name, ur.role_id, ur.user_role_id, ur.dac_id " + " FROM dacuser du "
+            + " LEFT JOIN user_role ur ON ur.user_id = du.dacuserid " + " LEFT JOIN roles r ON r.roleid = ur.role_id "
+            + " WHERE du.institution_id = :institutionId")
+    List<User> findUsersByInstitution(@Bind("institutionId") Integer institutionId);
```

Comment:
nit: break these up one line per query segment so its easier to read. 
```suggestion
    @SqlQuery("SELECT du.*, r.name, ur.role_id, ur.user_role_id, ur.dac_id "
            + " FROM dacuser du "
            + " LEFT JOIN user_role ur ON ur.user_id = du.dacuserid "
            + " LEFT JOIN roles r ON r.roleid = ur.role_id "
            + " WHERE du.institution_id = :institutionId")
    List<User> findUsersByInstitution(@Bind("institutionId") Integer institutionId);
```